dFusion/dFusion.rst

```diff
@@ -34,24 +34,25 @@ Due to the nature of fraud proofs, the system sets a crypto-economic incentive t
 State stored in the smart contract
 ==================================
 
-For each account, ERC20 token balance are merklized in a Merkle tree with the sha-hash operation in the anchor smart contract.
-This "compressed" representation of all account balances is collision resistant and can thus be used to uniquely commit to the complete "uncompressed" state containing all balances explicitly. 
+For each account, ERC20 token balances are represted in a Merkle tree constructed by the sha-hash operation.
+This "compressed" representation of all account balances by the Merkle root - called the balance-state-hash - is collision resistant and can thus be used to uniquely commit to the complete "uncompressed" state containing all balances explicitly. 
```

Comment:
Suggestion: "... by the Merkle root, called the *balance state hash*, is collision..."

Reply:
Perhaps we can use the word "succinct" instead of compressed?